torch/nn/modules/sparse.py

```diff
@@ -28,6 +31,11 @@ class Embedding(Module):
         - Input: LongTensor `(N, W)`, N = mini-batch, W = number of indices to extract per mini-batch
         - Output: `(N, W, embedding_dim)`
 
+    Notes:
+        Keep in mind that only a limited number of optimizers support
```

Comment:
Please mention leaf Variables

Reply:
Sure, but should we mention it in the `Module` docs too? Since Embedding takes care of creating (leaf) weights Variable.

Reply:
Ah sorry. No, it doesn't apply to modules :)